packages/ast-position/lib/ast-position.js

```diff
@@ -0,0 +1,41 @@
+class AstPositionVisitor {
+  constructor(offset) {
+    this.offset = offset;
+  }
+  visitXMLElement(node) {
+    const openName = node.syntax.openName;
+    const closeName = node.syntax.closeName;
+    if (this.isOffsetInRange(openName)) {
+      this.astContext = { kind: "XMLElementOpenName", astNode: node };
+    }
+
+    if (this.isOffsetInRange(closeName)) {
```

Comment:
Use `else if` here (if you already found the range in the open tag you shouldn't check the close tag too)